CardScan/Classes/CreditCardUtils.swift

```diff
@@ -1,6 +1,25 @@
 import Foundation
 
 public struct CreditCardUtils {
+    static let cvcLengthAmericanExpress = 4
```

Comment:
are the cvc lengths getting used anywhere?

Reply:
no but i thought maybe we'd use them later?

Reply:
Maybe, but let's remove them for now and add them back when we need them.